src/utils/fetchArticles.js

```diff
@@ -0,0 +1,6 @@
+import axios from "axios";
+
+export default function fetchArticle() {
```

Comment:
It looks like you intend to split your api calls into different topics e.g. articles, comments, etc. This is fine but you ay also save yourself some file space if you create a baseUrl in the file and have all the api calls stored in here, named something more appropriate.